code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1098,6 +1098,22 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		src.updateUsrDialog()
 		return 0
 
+	//Fill the machine with the pill collector!
+	if(istype(O, /obj/item/weapon/storage/bag/chem))

```

Comment:
I considered that, but I thought perhaps there was a reason they were casting the item inside the bag as grown.